### DISCUSSION TITLE
Parameterize loader return value with storybook args

### DISCUSSION THREAD

**Discussion by tschmidt01** [JesusTheHun/storybook-addon-remix-react-router#72] [Q&A]
When using this plugin with remix, we cannot execute the real loader as there is no Remix server running so I need to essential mock the response value of the loader when defining a story:

```
export const SomeStory = {
    parameters: {
        reactRouter: reactRouterParameters({
            routing: {
                id: 'root',
                loader: () => {
                    return { someProp: 'someValue' };
                },
            },
        }),
    },
};
```

Is there any way to parameterize the loader response using Storybook args?

**Comment by JesusTheHun** [comment]
Hi @tschmidt01 👋

At the moment no there is no way to do that.
But I think  `reactRouterParameters` could accept a function that receive the story args as the first parameter. Although SB has a long standing issue with story args and React, so I'm not sure it's feasible.

Are you willing to try and submit a PR for this ?
